PVSSpringBoot/src/main/java/com/example/PVSSpringBoot/config/SecurityConfiguration.java

```diff
@@ -36,7 +36,8 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
                 .and()
                 .authenticationProvider(authenticationProvider)
-                .addFilterBefore(jwtAuthFilter, UsernamePasswordAuthenticationFilter.class);
+                .addFilterBefore(jwtAuthFilter, UsernamePasswordAuthenticationFilter.class)
+                .cors();
```

Comment:
what does the .cors() method do?

Reply:
without it the headers of the request are removed and the token is removed then the request has no access to the controller.